code/controllers/subsystem/pai.dm

```diff
@@ -58,13 +72,13 @@ SUBSYSTEM_DEF(pai)
 				candidate.savefile_load(usr)
 				//In case people have saved unsanitized stuff.
 				if(candidate.name)
-					candidate.name = copytext_char(candidate.name,1,MAX_NAME_LEN)
+					candidate.name = copytext_char(sanitize(candidate.name),1,MAX_NAME_LEN)
 				if(candidate.description)
-					candidate.description = copytext_char(candidate.description,1,MAX_MESSAGE_LEN)
+					candidate.description = copytext_char(sanitize(candidate.description),1,MAX_MESSAGE_LEN)
 				if(candidate.role)
-					candidate.role = copytext_char(candidate.role,1,MAX_MESSAGE_LEN)
+					candidate.role = copytext_char(sanitize(candidate.role),1,MAX_MESSAGE_LEN)
 				if(candidate.comments)
-					candidate.comments = copytext_char(candidate.comments,1,MAX_MESSAGE_LEN)
+					candidate.comments = copytext_char(sanitize(candidate.comments),1,MAX_MESSAGE_LEN)
```

Comment:
Revert this to be the same as on the left, this doesn't need to be sanitized. I changed that in a past PR, so you shouldn't be reverting it back to that.